ts/Dashboard/Dashboard.ts

```diff
@@ -28,10 +28,12 @@ import Bindings from './Actions/Bindings.js';
 import Globals from './Globals.js';
 import EditMode from './EditMode/EditMode.js';
 import H from '../Core/Globals.js';
-const { doc } = H;
 import Layout from './Layout/Layout.js';
 import Serializable from './Serializable.js';
 import U from '../Core/Utilities.js';
+import Fullscreen from './EditMode/Fullscreen.js';
+
+const { doc } = H;
```

Comment:
We should use `window.document` and not the Highcharts globals. Or at least use Dashboard globals.